### DISCUSSION TITLE
Learn Python Cybersecurity

### DISCUSSION THREAD

**Discussion by Takzuin** [community/community#145063] [Discover: GitHub Best Practices]
To learn about python and cybersecurity or micropython with esp32...etc, where are the better books? like i see a lot but they were released like 5 to 10 years ago

**Comment by Takzuin** [comment]
Respond

**Comment by Takzuin** [comment]
wahapen

**Comment by Takzuin** [comment]
Response #2

**Comment by Evinrrr** [comment]
For Python, "Python Crash Course" is a great choice for beginners. For cybersecurity, "Hacking: The Art of Exploitation" is quite well - known. As for MicroPython with ESP32, "MicroPython Cookbook" is recommended. These books are relatively recent and very useful.
